iothub/service/src/FeedbackRecord.cs

```diff
@@ -7,25 +7,51 @@ namespace Microsoft.Azure.Devices
     using Newtonsoft.Json;
 
     /// <summary>
-    /// Contains properties of an individual feedback record. 
+    /// Contains properties of an individual feedback record.
     /// </summary>
     public class FeedbackRecord
     {
+        /// <summary>
+        /// The MessageId of the cloud-to-device message to which this feedback information relates.
+        /// </summary>
         [JsonProperty(PropertyName = "originalMessageId", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string OriginalMessageId { get; set; }
 
+        /// <summary>
+        /// String values for StatusCode.
+        /// </summary>
         [JsonProperty(PropertyName = "description", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string Description { get; set; }
 
+        /// <summary>
+        /// The DeviceGenerationId of the target device of the cloud-to-device message to which this piece of feedback relates.
+        /// </summary>
         [JsonProperty(PropertyName = "deviceGenerationId", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string DeviceGenerationId { get; set; }
 
+        /// <summary>
+        /// The DeviceId of the target device of the cloud-to-device message to which this piece of feedback relates.
+        /// </summary>
         [JsonProperty(PropertyName = "deviceId", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string DeviceId { get; set; }
 
+        /// <summary>
+        /// Date and time that indicates when the outcome of the message happened (for example, the hub received the feedback message or the original message expired).
+        /// </summary>
         [JsonProperty(PropertyName = "enqueuedTimeUtc", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public DateTime EnqueuedTimeUtc { get; set; }
 
+        /// <summary>
+        /// The status of feedback messages that are generated by the IoT Hub.
+        /// </summary>
+        /// <remarks>
+        /// Possible values are:
+        /// Success
```

Comment:
Is there any way to make this show up as an actual list?

Reply:
Found a way and updated